### PR TITLE
MIB-1 Manually adjust rowsperstrip parameter when saving mibitiffs

### PR DESCRIPTION
**Pull request by murataksoy**
See https://ionpath.atlassian.net/browse/MIB-1 for details

### CODE REVIEW

File: mibidata/tiff.py [hunk 1]
```diff
@@ -164,18 +164,22 @@ def write(filename, image, sed=None, optical=None, ranges=None,
                     'channel.mass': int(image.masses[i]),
                     'channel.target': image.targets[i],
                 })
-                page_name = (
-                    285, 's', 0, '{} ({})'.format(image.targets[i].encode(),
-                                                  image.masses[i])
-                )
+
+                # Converting to bytes string to support non-ascii characters
```
Comment (DanaCase) [inline mibidata/tiff.py:168]: Hum shouldn't we specify this tag as a byte type instead of a string?

File: mibidata/tiff.py [hunk 2]
```diff
@@ -164,18 +164,22 @@ def write(filename, image, sed=None, optical=None, ranges=None,
...
+                page_name = (285, 's', 0, page_name_string)
                 min_value = (340, range_dtype, 1, ranges[i][0])
                 max_value = (341, range_dtype, 1, ranges[i][1])
                 page_tags = coordinates + [page_name, min_value, max_value]
 
                 infile.write(
                     to_save[:, :, i], compress=6, resolution=resolution,
                     extratags=page_tags, metadata=metadata, datetime=image.date,
-                    software=SOFTWARE_VERSION)
+                    software=SOFTWARE_VERSION, rowsperstrip=to_save.shape[0])
```
Comment (DanaCase) [inline mibidata/tiff.py:182]: Can you please include a comment that explains this parameter?

File: mibidata/tiff.py [hunk 3]
```diff
@@ -211,8 +217,12 @@ def write(filename, image, sed=None, optical=None, ranges=None,
                 'channel.mass': int(image.masses[i]),
                 'channel.target': image.targets[i],
             })
-            page_name = (285, 's', 0, '{} ({})'.format(
-                image.targets[i].encode(), image.masses[i]))
+            # Converting to bytes string to support non-ascii characters
```
Comment (DanaCase) [inline mibidata/tiff.py:220]: Since we repeat this code twice, should it be in a function?

**Comment by DanaCase** [comment]
I read in a tiff and wrote it on this branch and on master.
Can confirm that the tiff is importable to bioformats on this branch but not on master.

File: mibidata/tiff.py (hunk 3 above)
Comment (murataksoy) [reply mibidata/tiff.py:220]: Done

**Review by DanaCase — COMMENTED** [review]
Looks good.  Thanks Murat!

**Review by DanaCase — APPROVED** [review]

**Pull request merged**